Fix compilation with cython 3

mvdbeek opened the pull request.

mvdbeek committed:
Fix integer division

mvdbeek committed:
Fix print statement syntax

mvdbeek committed:
Fix imports

mvdbeek committed:
Extend tests to include python 3.11

mvdbeek committed:
Include Python 3.11 as officially supported

mvdbeek committed:
Run black

nsoranzo commented on scripts/bed_count_by_interval.py:
```diff
@@ -17,7 +17,11 @@
+        chrom,
+        start,
+        end,
+    ) = (
```
Maybe break this down into 3 separate assignments?

nsoranzo commented on scripts/bed_count_overlapping.py:
```diff
@@ -17,7 +17,11 @@
+        chrom,
+        start,
+        end,
+    ) = (
```
Same.

nsoranzo committed:
Lint with black and isort in CI workflow

Also:
- Don't lint again in the test step of the CI workflow
- Fix import order
- Replace some star imports
- Remove some trailing whitespaces

nsoranzo committed:
Fixup

mvdbeek committed:
Include pxd files in source dist

mvdbeek commented:
Hmm, the pypy builds are taking a very long time ...

nsoranzo committed:
Drop PyPy 3.8 wheels

nsoranzo commented:
> Hmm, the pypy builds are taking a very long time ...

Yep, PyPy v7.3.12 added Python 3.10 and dropped 3.8, so I've just pushed a commit to drop PyPy 3.8 wheels.

nsoranzo commented:
Thanks @mvdbeek !

Should we make a new release?

mvdbeek commented:
Yes please! I can (probably) do the release notes and tagging.

mvdbeek commented:
Nope, doesn't look like I have the permissions to do that.

nsoranzo commented:
No problem, I am doing it now, it's very lightweight for this repo.

nsoranzo commented:
New release v0.10.0 is on PyPI.
